Lib/test/test_importlib/test_metadata_api.py

```diff
@@ -17,6 +18,13 @@
 )
 
 
+@contextlib.contextmanager
+def suppress_known_deprecation():
+    with warnings.catch_warnings(record=True) as ctx:
+        warnings.simplefilter('default')
```

Comment:
Could be more specific with category=DeprecationWarning.

Reply:
I was tempted, now convinced. Will do that in _metadata to inherit here.

Reply:
Added in python/importlib_metadata@2678a5e5af2.